src/Common/LanguageModel.cs

```diff
@@ -3,11 +3,14 @@
 // Copyright (c) GlobalLink Vasont. All rights reserved.
 // </copyright>
 //-----------------------------------------------------------------------
+using System;
+
 namespace Vasont.Inspire.Models.Common
 {
     /// <summary>
     /// This class represents a supported language definition for the application.
     /// </summary>
+    [Obsolete("Use local DTOs", true)]
```

Comment:
This message is fine so long as the local DTO is the same exact name, otherwise I would specify the name.

Reply:
Same comment for all the other Obsolete messages.

Reply:
@EricRPG has done this in the past. It may be useful going forward, but mostly all migrated models are simply named blablaDto from blablaModel. Here I am also just catching up from others recent migrations that don't have access to this repo.